Server/JokesOnYou.Web.Api/Repositories/TagRepository.cs

```diff
@@ -36,13 +62,12 @@ public TagRepository(DataContext context, IMapper mapper)
                    join userJokeTags in _context.UserJokeTags
```

Comment:
Can we convert this to method syntax as that has been used across the project?

Reply:
Yes. 
Should i have to change it?